src/common/transformations/include/ngraph_ops/nms_ie_internal.hpp

```diff
@@ -53,9 +76,14 @@ class TRANSFORMATIONS_API NonMaxSuppressionIEInternal : public Op {
     bool m_sort_result_descending = true;
     element::Type m_output_type;
     element::Type m_scores_output_type;
+    bool isSoftNms() {
```

Comment:
bool isSoftNms() 
shouldn't be  bool isSoftNms() const?

Reply:
yes, but code is already reverted)